cmplibrary/src/main/java/com/sourcepoint/cmplibrary/core/web/ConsentWebView.kt

```diff
@@ -107,8 +109,8 @@ internal class ConsentWebView(
         consent: String?,
     ): Either<Boolean> = check {
 
-        if (connectionManager.isConnected.not()) throw NoInternetConnectionException(
-            description = "No internet connection"
+        if (context.isInternetConnected().not()) throw NoInternetConnectionException(
+            description = context.getString(R.string.exception_no_internet_connection_text)
```

Comment:
@bohdan-go-wombat could you move the description to be a static (or regular) field of `NoInternetConnectionException` ? This way we avoid having a dependency on `R` in our code. Remember this is likely to get replaced by the core module and the core module probably won't have access to R.

Reply:
@andresilveirah I guess you are correct on this one, since we are not going to translate this description into any other language

although, it is a good practise to use string constants from the string resources rather than hardcoding it in a bunch of places (hardcoding makes it harder to change the value of this string in the future)